.github/workflows/autopr_bugfix.yml

```diff
@@ -0,0 +1,51 @@
+name: Cherrypick bugfixes to release branch
+on:
+  push:
+    branches:
+      - dev
+jobs:
+  # Check if the PR is a bugfix
+  check_labels:
+    runs-on: ubuntu-latest
+    name: Check PR labels action step
+    # Make the output of this job available to other jobs
+    outputs: 
+      result: ${{steps.check_pr_labels.outputs.result}}
+    steps:
+    - id: check_pr_labels
+      uses: 3pointer/check-pr-labels-on-push-action@v1.0.3
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        labels: '["bug"]'
+    # Print the result to the log
+    - name: See result
+      run: echo "${{ steps.check_pr_labels.outputs.result }}"
+  release_pull_request:
+    needs: check_labels
+    # Only run this step if the code was a bugfix
+    if: contains(needs.check_labels.outputs.result, 'true')
+    runs-on: ubuntu-latest
+    name: release_pull_request
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2.2.0
+      # Fetch all info including branches + tags
+      with:
+        fetch-depth: 0
+    - name: 'Get Previous tag'
+      id: get_latest_tag
+      uses: "WyriHaximus/github-action-get-previous-tag@v1"
+    - id: get_short_version
+      run: |
+        # Get the major / minor version without patch info i.e. 6.0 from 6.0.1
+        major_minor=$(echo ${{steps.get_latest_tag.outputs.tag}} | cut -d '.' -f 1,2)
+        echo "::set-output name=major_minor::$major_minor"
+    - name: Create PR to branch
+      uses: adamtharani/github-action-cherry-pick@master
+      with:
+        # PR to the latest feature release.  Merges are enabled
+        pr_branch: ${{steps.get_short_version.outputs.major_minor}}
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        DRY_RUN: false
```

Comment:
Do we want to turn this on with DRY_RUN: true for a little while and then flip it all the way on?